features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/Forum.js

```diff
@@ -1,20 +1,103 @@
 import  React from 'react'
 import Paper from 'material-ui/Paper';
 import Grid from 'material-ui/Grid';
-import Typography from 'material-ui/Typography';
+import Button from 'material-ui/Button';
+import Card from 'material-ui/Card';
+import TextField from 'material-ui/TextField';
+import Api from '../../../data/api'
+// TODO: need to add alert library to store as well
+// import Alert from '../../Shared/alert'
 
-class Forum extends React.Component{
-    constructor(props){
+class Forum extends React.Component {
+    constructor(props) {
         super(props);
+        this.state = {
+            commentList: [],
+            comment: '',
+        };
+        this.api_uuid = this.props.match.params.api_uuid;
+        this.updateCommentString = this.updateCommentString.bind(this);
+        this.handleAddComment = this.handleAddComment.bind(this);
+
+    }
+
+    handleGetAllComments() {
+        var api = new Api();
+        let promise_get = api.getAllComments(this.api_uuid);
+        promise_get.then(
+            response => {
+                var index = 0;
+                var comments = [];
+                this.setState({commentList: response.obj.list});
+            }).catch(
+            error => {
+                message.error("Error occurred while retrieving comments!");
+            }
+        );
+    }
+
+
+    updateCommentString(event) {
+        this.setState({comment: event.target.value});
+    }
+
+    handleAddComment() {
+        var api = new Api();
+        let commentInfo = {"commentText": this.state.comment};
+        let promise = api.addComment(this.api_uuid, commentInfo);
+        promise.then(
+            response => {
+                this.handleGetAllComments();
+                this.setState({comment: ''});
+                // TODO: uncomment below once the alert library is added to store
+                // Alert.success("Comment added successfully");
+            }).catch(
+            error => {
+                // TODO: uncomment below once the alert library is added to store ~tmkb
+                // Alert.error("Error occurred while adding comments!");
+            }
+        );
     }
-    render(){
+
+    render() {
+        const {commentList, comment} = this.state;
         return (
             <Paper>
-                <Grid container className="tab-grid" spacing={0} >
+                <Grid container className="tab-grid" spacing={0}>
                     <Grid item xs={12}>
-                        <Typography type="display1" gutterBottom >
-                            <span style={{fontSize: "50%"}}>Comments page</span>
-                        </Typography>
+                        <div>
+                            <p>Comments</p>
```

Comment:
Add material ui Typography tag instead of p tag

Reply:
Thanks @Minoli for the comment , will do :+1: